easybuild/easyblocks/c/cp2k.py

```diff
@@ -668,9 +668,15 @@ def build_step(self):
         run_cmd(cmd + " clean", log_all=True, simple=True, log_output=True)
 
         # build and install
+        # compile regularly first with the default make target
+        # and only then build the library
+        run_cmd(cmd, log_all=True, simple=True, log_output=True)
```

Comment:
@jmbuhr This should be `cmd + " all"`, to ensure it does the same thing as before?

Reply:
The installation instructions now just say `make -j N ARCH=architecture VERSION=version` (https://github.com/cp2k/cp2k/blob/master/INSTALL.md#3a-arch-files), which is equivalent to `all` as the default target (https://github.com/cp2k/cp2k/blob/2f0c29e7cf8e6f46ad70372cd31142c44769af8f/Makefile#L39). But this also means keeping `all` here doesn't hurt.